assembl/static2/js/app/components/navbar/navbar.jsx

```diff
@@ -0,0 +1,189 @@
+import FlatNavbar from './FlatNavbar';
+import BurgerNavbar from './BurgerNavbar';
+import { APP_CONTAINER_MAX_WIDTH, APP_CONTAINER_PADDING } from '../../constants';
+import { displayModal } from '../../utils/utilityManager';
+import { getDiscussionSlug } from '../../utils/globalFunctions';
+
+const filterSection = ({ sectionType }, { hasResourcesCenter, hasSyntheses }) => {
+  switch (sectionType) {
+  case 'RESOURCES_CENTER':
+    return hasResourcesCenter;
+  case 'SYNTHESES':
+    return hasSyntheses;
+  case 'ADMINISTRATION':
+    return connectedUserIsAdmin();
+  case 'HOMEPAGE':
+    return false;
+  default:
+    return true;
+  }
+};
+
+const sectionFilter = (options = {}) => {
+  return (section) => {
+    return filterSection(section, options);
+  };
+};
+
+const sectionKey = ({ sectionType, url }) => {
+  return sectionType === 'CUSTOM' ? `${sectionType}-${url}` : sectionType;
+};
+
+const constantToCamelCase = (string) => {
+  return string.toLowerCase().replace(/_[a-z]/g, (match) => {
+    return match[1].toUpperCase();
+  });
+};
+
+const sectionSlug = (sectionType) => {
+  return constantToCamelCase(sectionType === 'HOMEPAGE' ? 'HOME' : sectionType);
+};
+
+const sectionURL = ({ sectionType, url }, slug, phaseIdentifier) => {
+  return url || `${get(sectionSlug(sectionType), { slug: slug, phase: phaseIdentifier })}`;
+};
+
+const SectionLink = ({ section, options: { slug, currentPhaseIdentifier } }) => {
+  const { title } = section;
+  return (
+    <Link
+      to={sectionURL(section, slug, currentPhaseIdentifier)}
+      className="navbar-menu-item pointer"
+      activeClassName="active"
+      data-text={title}
+    >
+      {title}
+    </Link>
+  );
+};
+
+const createDisplayModal = ({ debate, i18n }) => {
+  return () => {
+    const slug = { slug: getDiscussionSlug() };
+    const { timeline } = debate.debateData;
+    const { locale } = i18n;
+    const currentPhaseIdentifier = getCurrentPhaseIdentifier(timeline);
+    const phaseName = getPhaseName(timeline, currentPhaseIdentifier, locale).toLowerCase();
+    const body = <Translate value="redirectToV1" phaseName={phaseName} />;
+    const button = { link: get('oldDebate', slug), label: I18n.t('home.accessButton'), internalLink: false };
+    displayModal(null, body, true, null, button, true);
+    setTimeout(() => {
+      window.location = get('oldDebate', slug);
+    }, 6000);
+  };
+};
+
+const mapDebateSectionToElement = (debateSection, options) => {
+  const { title } = debateSection;
+  const { phaseContext, displayDebateModal } = options;
+  const key = sectionKey(debateSection);
+  switch (phaseContext) {
+  case 'modal':
+    return (
+      <div key={key} onClick={displayDebateModal} className="navbar-menu-item pointer" data-text={title}>
+        {title}
+      </div>
+    );
+  case 'old':
+    return (
+      <a key={key} className="navbar-menu-item pointer" href={get('oldDebate', { slug: options.slug })} data-text={title}>
+        {title}{' '}
```

Comment:
Why the space? Also, it could be `{'${title} '}`.

Reply:
bad eslint, bad!